src/host/ledger.h

```diff
@@ -132,14 +142,23 @@ namespace asynchost
     bool completed = false;
     bool committed = false;
 
+    bool recovery = false;
+
   public:
     // Used when creating a new (empty) ledger file
-    LedgerFile(const std::string& dir, size_t start_idx) :
+    LedgerFile(const fs::path& dir, size_t start_idx, bool recovery = false) :
```

Comment:
A `LedgerFile` can now be marked as `recovery` when created (this is done by the higher-level `Ledger` object). The file name will automatically be suffixed with `.recovery` (even when the file is committed, i.e. `ledger_x-y.committed.recovery`) and ignored/deleted by other nodes on startup. Recovery ledger files are automatically renamed (i.e. `.recovery` suffix dropped) once the service is open.